src/main/fc/fc_msp.c

```diff
@@ -3318,6 +3365,49 @@ static mspResult_e mspFcProcessInCommand(uint16_t cmdMSP, sbuf_t *src)
         gpsUbloxSendCommand(src->ptr, dataSize, 0);
         break;
 
+#if defined(USE_GEOZONE) && defined (USE_GPS)
```

Comment:
probably don't need to check for USE_GPS here. Maybe you should add an #error if USE_GPS is not defined in the geozone headers instead, to simplify the checks everywhere.